package.json

```
@@ -7,7 +7,7 @@
   "types": "mdi.d.ts",
   "sideEffects": false,
   "scripts": {
-    "build": "npm update && npm install && npm run buildjs && npm run es5 && npm run commonjs",
+    "build": "npm run buildjs && npm run es5 && npm run commonjs",
     "buildjs": "node build.js",
     "es5": "tsc -d mdi.ts --target es5 --module es2015",
     "commonjs": "tsc -d mdi.ts --outDir commonjs",
```

With the scripts like this, we ran into errors when trying to build. With the following changes, it seemed to work well:

```suggestion
    "es5": "ts-node mdi.ts --target ESNext --module CommonJS",
    "commonjs": "ts-node mdi.ts --outDir commonjs --module CommonJS",
```